.github/workflows/release.yml

```diff
@@ -16,9 +16,12 @@ jobs:
       - name: "Prune docker"
```

Comment:
This diff is quite confusing, you can browse the file [here](https://github.com/faasm/faasm/blob/sgxless/.github/workflows/release.yml).